src/html.cr

```diff
@@ -36,10 +36,33 @@ module HTML
   # HTML.escape("Crystal & You", io) # => nil
   # io.to_s                          # => "Crystal &amp; You"
   # ```
+  @[AlwaysInline]
```

Comment:
Up for discussion: I don't think this annotation is really necessary. Pretty sure LLVM will happily inline this call even without being told.

We typically use this annotation to indicate when inlining is *absolutely* necessary for correct semantics and not for every high-level method delegation (where it obviously makes sense, but isn't mandatory).